azure/kubernetes/aks-single-region/variables.tf

```diff
@@ -16,6 +16,31 @@ variable "resource_prefix" {
   default     = "camunda"
 }
 
+variable "kubernetes_version" {
+  description = "Kubernetes version to use for the AKS cluster"
+  type        = string
+  default     = "1.28.5"
```

Comment:
would probably bump to a higher version as in e.g. EKS we're on 1.32 atm.

You can also think about how this should be updated via Renovate.